src/test/java/eu/dissco/core/digitalspecimenprocessor/repository/DigitalSpecimenRepositoryIT.java

```diff
@@ -25,6 +29,8 @@ class DigitalSpecimenRepositoryIT extends BaseRepositoryIT {
 
   private DigitalSpecimenRepository repository;
 
+  private MockedStatic<Clock> clockMock;
```

Comment:
If we use .isAfter then we can remove the MockedStatic completly right? Or is still used somewhere

Reply:
Removed